docs/source/roadmap.rst

```diff
@@ -58,13 +59,20 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v3.7
+
+* **Action Clone:** Actions can be cloned via API, CLI, and web UI.
+* **Datastore RBAC:** Accessing datastore Key Value pairs can be restricted with RBAC.
+* **Action-Alias Testing:** Use ``st2 action-alias test <message string>`` to easily test action aliase matching and result formatting.
+* **Expanded Garbage Collection:** The garbage collector cleans up many more resources now.
```

Comment:
How about the https://stackstorm.com/2022/05/03/override-default-resource-status-new-feature-in-stackstorm-3-7-0/ ?

Reply:
oh yes!

Reply:
Between the major changes, there's also RockyLinux migration as we replace deprecated EL8 with the RockyLinux.

Reply:
added both of these. What else did I miss?